src/Microsoft.Framework.PackageManager/Program.cs

```diff
@@ -61,7 +61,9 @@ public int Main(string[] args)
             {
                 c.Description = "Restore packages";
 
-                var argRoot = c.Argument("[root]", "Root of all projects to restore. It can be a directory, a project.json, or a global.json.");
+                var argRoot = c.Argument("[root]",
+                    "Root of projects to restore. A value can be a path to directory/project.json/global.json. Accepts multiple values.",
+                    multipleValues: true);
```

Comment:
@Eilon , I bet you have a better idea about help text of the argument here. This argument can have multiple values:
```
dnu restore <path1> <path2> <path3>
```
The three paths are all values of the argument `[root]`